test/pylib/minio_server.py

```diff
@@ -52,6 +53,41 @@ def log_to_file(self, str):
         self.log_file.write('\n'.encode())
         self.log_file.flush()
 
+    def _anonymous_public_policy(self):
+        # the public policy does not allow accessing the tagging,
+        # so let's add them manually
```

Comment:
The structure below has a bunch of weird details that I'm guessing you copied from somewhere. Might it be possible and perhaps better to write code that copies the existing policy file and *add* the Tagging actions instead of creating the whole thing from scratch?

I'm worried that a year from now, this hack will stop working because minio's "policy file" will have changed and will need additional things that you didn't copy.

But on the other hand, it's not really a big problem, if you continue to maintain this script.